MapboxSearch/sdk/src/main/java/com/mapbox/search/SearchResultMetadata.kt

```diff
@@ -135,7 +313,32 @@ public class SearchResultMetadata internal constructor(
             otherPhotos?.map { it.mapToCore() },
             cpsJson,
             parking?.mapToCore(),
-            metadata
```

Comment:
Is SearchResultMetadata something publicly available and was released before? Is moving the metadata position a breaking change?

Reply:
This is an internal constructor so no consumer impact.